.github/workflows/wheels.yml

```diff
@@ -134,6 +153,12 @@ jobs:
         with:
           python-version: '3.9'  # update once build dependencies are available
 
+      - name: Set pip extra index
+        if: github.event_name == 'schedule'
+        env:
+          PIP_EXTRA_INDEX: https://pypi.anaconda.org/scipy-wheels-nightly/simple
```

Comment:
According to [GitHub docs](https://docs.github.com/en/actions/learn-github-actions/environment-variables#about-environment-variables), I think this needs to be defined one level up: [jobs.<job_id>.env](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idenv).

Reply:
Also, I suspect we need `PIP_PRE=1` to actually download the pre-release builds from the nightly index.